src/com/yaji/Harness.java

```diff
@@ -0,0 +1,46 @@
+package com.yaji;
+
+import java.util.Date;
+
+public class Harness {
+        private Date purchaseDate, lastDate;
+        private double  size;
+        private int gender;
+        private String brand;
+
+
+        public Harness(Date purchase_date, Date last_date, String harness_brand, double harness_size, int Gender){
```

Comment:
We are now getting into the details, but better get used to the right conventions as early as possible: parameters are camelCase starting with lower case letters, just like class fields. In order to avoid name collisions in your scope, you can write `this.gender = gender`